src/test/java/uk/gov/digital/ho/pttg/application/namematching/candidates/NamesWithAbbreviationSpaceCombinationsAbbreviatedNamesFunctionsTest.java

```diff
@@ -6,10 +6,10 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static uk.gov.digital.ho.pttg.application.namematching.candidates.AbbreviatedNamesFunctions.doesNotContainAbbreviatedNames;
 
-public class AbbreviatedAbbreviatedNamesWithFullStopSpaceCombinationsFunctionsContainFullStopSpaceTest {
+public class NamesWithAbbreviationSpaceCombinationsAbbreviatedNamesFunctionsTest {
 
     @Test
-    public void doesNotContainFullStopSpaceBetweenNames_emptyString_returnTrue() {
+    public void doesNotContainAbbreviationSpaceBetweenNames_emptyString_returnTrue() {
```

Comment:
I think the names of these tests have started getting out of step with what's being tested, although I think the problem probably started before you picked up this ticket. But as you've been working here, now is a good time to sort it.

The Osherove method which we try to follow http://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html is
UnitOfWork_StateUnderTest_ExpectedBehaviour

So for this test, for example, it should probably be doesNotContainAbbreviatedNames_emptyString_returnTrue()

Reply:
Resolved this by renaming each test by beginning with the unit of work it is testing. In this case it is doesNotContainAbbreviatedNames.